src/commands/lambda/__tests__/flare.test.ts

```diff
@@ -986,4 +978,8 @@ describe('lambda flare', () => {
       expect(result).toEqual(expectedFiles)
     })
   })
+
+  test('summarizeConfig', () => {
+    expect(summarizeConfig(MOCK_LAMBDA_CONFIG)).toMatchSnapshot()
+  })
```

Comment:
Is this test necessary? This is already being tested by snapshots in other tests, but I still think it doesn't hurt to have this test

Reply:
Yeah I think it doesn't hurt